src/main/lombok/org/heigit/ohsome/ohsomeapi/controller/dataaggregation/CountController.java

```diff
@@ -337,6 +346,9 @@ public Response countRatio(HttpServletRequest servletRequest, HttpServletRespons
           defaultValue = "", paramType = "query", dataType = "string", required = false),
       @ApiImplicitParam(name = "values2", value = ParameterDescriptions.DEPRECATED_USE_FILTER2,
           defaultValue = "", paramType = "query", dataType = "string", required = false),
+      @ApiImplicitParam(name = "filter2", value = ParameterDescriptions.FILTER,
```

Comment:
I guess you wanted to use here `filter` not `filter2` right?

Reply:
good catch. yes that was the intention